pkg/api/server/cel/view/records.go

```diff
@@ -0,0 +1,38 @@
+package view
+
+import (
+	"github.com/google/cel-go/cel"
+	"github.com/tektoncd/results/pkg/api/server/cel2sql"
+)
+
+func NewRecordsView() (*cel2sql.View, error) {
+	view := &cel2sql.View{
+		Constants: map[string]cel2sql.Constant{},
+		Fields: map[string]cel2sql.Field{
+			"parent": {
+				CELType: cel.StringType,
+				SQL:     `parent`,
```

Comment:
Is this field the column name in the corresponding table? If so, I'd rename it to make the intent clearer.

Reply:
Not necessarily will be a column.. it can be an operation like the concatenation of multiple columns or values